content/reference-architectures/cold-start.md

```diff
@@ -0,0 +1,627 @@
+mfa_serial=arn:aws:iam::323330167063:mfa/admin@cloudposse.co
+source_profile=cpco
+
+# This profile is required by aws-vault where it stores the Access Key ID and Secret Access Key as explained in Prerequisites
+[profile cpco]
+```
+{{% /dialog %}}
+
+## Provision `root-dns` Project In `root` To Create `parent` And `root` DNS Zones
+
+Now we provision DNS for the `root` account, but without the `testing` Name Servers yet.
+
+They will be provisioned later and then we'll come back to add them.
+
+{{% dialog type="code-block" icon="fa fa-code" title="Provision `root-dns` Project In `root` To Create `parent` And `root` DNS Zones" %}}
+```shell
+cd root-dns
+init-terraform
+
+terraform apply -target=aws_route53_zone.parent_dns_zone -target=aws_route53_record.parent_dns_zone_soa \
+    -target=aws_route53_zone.root_dns_zone -target=aws_route53_record.root_dns_zone_soa \
+    -target=aws_route53_record.root_dns_zone_ns
+```
+{{% /dialog %}}
+
+
+You should see Terraform output similar to this:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Terraform output" %}}
+```
+parent_name_servers = [
+    ns-1154.awsdns-16.org,
+    ns-1867.awsdns-41.co.uk,
+    ns-54.awsdns-06.com,
+    ns-704.awsdns-24.net
+]
+parent_zone_id = ZTNG19A4IP6XF
+root_name_servers = [
+    ns-1232.awsdns-26.org,
+    ns-1785.awsdns-31.co.uk,
+    ns-458.awsdns-57.com,
+    ns-650.awsdns-17.net
+]
+root_zone_id = Z3AZCXQQNZKZ7E
+```
+{{% /dialog %}}
+
+{{% dialog type="info" icon="fa-info-circle" title="Note" %}}
+If you did not buy the `parent` domain from Route53, you need to take the `parent` Name Servers from the Terraform output and update them in the registrar.
+{{% /dialog %}}
+
+
+## Build And Start Geodesic Module For `testing`
+
+Open a new terminal window and execute the following commands:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Build And Start Geodesic Module For `testing`" %}}
+```shell
+cd ~/testing.cloudposse.co
+
+# Initialize the project's build-harness
+make init
+
+# Build docker image
+make docker/build
+
+# Install the wrapper shell
+make install
+
+# Run the shell
+testing.cloudposse.co
+
+# Login to AWS as the admin user with your MFA device
+assume-role
+```
+{{% /dialog %}}
+
+You should see the `Docker` image built, `geodesic` shell started, and after you run the `assume-role` command, you will be logged in to the `testing` account:
+
+```
+Enter passphrase for /localhost/.ssh/id_rsa:
+Identity added: /localhost/.ssh/id_rsa (/localhost/.ssh/id_rsa)
+-> Run 'assume-role' to login to AWS
+ ⧉  testing.cloudposse.co
+❌   (none) ~ ➤  assume-role
+Enter passphrase to unlock /conf/.awsvault/keys/:
+* Assumed role arn:aws:iam::126450723953:role/OrganizationAccountAccessRole
+```
+
+
+## Provision `tfstate-backend` Project For `testing`
+
+Execute this sequence of steps in the `testing` geodesic session:
+
+{{% dialog type="code-block" icon="fa fa-code" title="Provision `tfstate-backend` Project For `testing`" %}}
```

Comment:
This should say to run the script that @goruha wrote

Reply:
will do it in the next PR after the scripts PR gets reviewed, tested and approved.
otherwise, too many changes in this PR